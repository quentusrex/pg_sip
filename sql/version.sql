CREATE EXTENSION pg_sip;
select pg_sip_version();
