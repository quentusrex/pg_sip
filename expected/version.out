CREATE EXTENSION pg_sip;
select pg_sip_version();
 pg_sip_version 
----------------
 0.0.1
(1 row)

